server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRestoreTask.java

```diff
@@ -56,6 +60,18 @@ public String getUser() {
             return user.asString();
         }
 
+        public String getRequestedQuery() {
+            return requestedQuery;
+        }
+
+        public Optional<String> getParsingErrorCause() {
```

Comment:
If there is a parsing error we probably should not create the task in the first place.

So this field in taskInfo makes little sense.